org.gnome.Notes.json

```diff
@@ -0,0 +1,145 @@
+{
+    "app-id" : "org.gnome.Notes",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.32",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "bijiben",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=host",
```

Comment:
I'm not a fan of such wide permissions. It should use xdg-desktop-portal instead.

Reply:
That's what we have for org.gnome.bijiben so far. We should test it with narrow permission to make sure it works.
Can't we leave like this for this initial commit and then improve it upstream?

Reply:
We should test it without now (you can just do that with `flatpak run --nofilesystem=host org.gnome.bijiben` (or org.gnome.Notes) and then file issues as required upstream. I won't block addition on this, but we do need to make a bit of an effort.

Reply:
I just did. With nofilesystem, there is no way to import notes. We can imrprove that to be read only though.

Reply:
That would be cool, does notes import need access to a folder or is it a single file (or selection of files) at a time process? (if it's a folder then the files portal just won't work so read only is the best we can do for now)